src/components/common/InputField.jsx

```diff
@@ -16,6 +16,7 @@ const InputField = (props) => {
     id,
     checkConfirm,
     isButtonClicked,
+    autoComplete,
```

Comment:
why use that?

Reply:
We are using autocomplete attributes in the input fields. Because this file not using the <form> tag to toggle the autocomplete. autocomplete lets web developers specify what if any permission the user agent has to provide automated assistance in filling out form field values, as well as guidance to the browser as to the type of information expected in the field.